packages/internal/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@agoric/internal",
+  "version": "0.1.0",
+  "description": "Externally unsupported utilities internal to agoric-sdk",
+  "type": "module",
+  "main": "src/index.js",
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "scripts": {
+    "build": "exit 0",
+    "test": "ava",
+    "test:nyc": "exit 0",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:eslint": "eslint .",
+    "lint:types": "tsc -p jsconfig.json"
+  },
+  "dependencies": {
+    "@endo/eventual-send": "^0.15.5",
+    "@endo/promise-kit": "^0.2.43"
+  },
+  "devDependencies": {
+    "@endo/init": "^0.5.43",
+    "ava": "^4.3.1"
+  },
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "files": [
```

Comment:
Need to include the src folder for publishing.

Consider using an `exports` field if you want to restrict deep imports.

Reply:
Yeah that's [my plan](https://github.com/Agoric/agoric-sdk/pull/6001#issuecomment-1221097283)

Reply:
You should be able to have a single exports for the main entrypoint for now:

```json
  "exports": {
    ".": "./src/index.js"
  }
```

Reply:
With that there are errors like,
```
Cannot find module '@agoric/internal' or its corresponding type declarations.
```

Btw, why would we need to export package.json?

Reply:
Separately, do you still have blocking request changes?

Reply:
Really? That's strange. The package.json I believe is necessary for the Endo bundler to work correctly. @kriskowal has the most experience on the exports field.